### DISCUSSION TITLE
PDF Output - way to list selected records on the same page

### DISCUSSION THREAD

**Discussion by GarethHumbel** [YetiForceCompany/YetiForceCRM#16166] [How to]
Guys, I am trying to print records from services module (Picture field and other three columns). Not sure where and how I can make it in continuous list (record under each other) Currently if I set field variable it prints each record on the other page.
How to?

**Comment by s2b-git** [comment]
Also struggling. Cannot figure out how to print services in form of two columns catalog in Z order

**Comment by ArniSamano** [comment]
$(recordsList : | | | ) working well, however it listing the same set of records twice for some reason. Also it not respecting "selected" records but whole set assigned to current user. How to trick it?

**Comment by mariuszkrzaczkowski** [comment]
you tried it
`$(custom : RecordsList)$`

**Reply by s2b-git** [reply]
yes but I need customize columns, so $(recordsList  only
I guess ArniSamano have the same thing

**Reply by mariuszkrzaczkowski** [reply]
set the columns in the custom view

**Reply by s2b-git** [reply]
Marius, $(custom : RecordsList)$ doesnt print images form records. Is there any trick on that?

Cheers!

**Comment by mariuszkrzaczkowski** [comment]
I added a fix that fixes the display of photos in "$ (custom: RecordsList) $" and in "$ (recordsList: Products) $" fixes filtering
https://github.com/YetiForceCompany/YetiForceCRM/commit/7a29a99dc913dc07722cab2f7362e0f1f18c4a50

**Reply by ArniSamano** [reply]
:-) Thanks!
